src/app/pages/LiquidityMining/components/RemoveLiquidityDialog/index.tsx

```diff
@@ -183,6 +184,7 @@ export const RemoveLiquidityDialog: React.FC<IRemoveLiquidityDialogProps> = ({
                 tx.loading || !valid || !canInteract || removeLiquidityLocked
               }
               className="tw-rounded-lg"
+              data-action-id="yieldFarm-withdrawal-button"
```

Comment:
```suggestion
              data-action-id="yieldFarm-liquidityModal-confirm"
```

Spec has been updated to use a shared attribute name for the confirm buttons on both add/remove liquidity modals